### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ilaflott**
too much stuff in base README.md - some of it better belongs in a CONTRIBUTING.md document. this pushes us more towards standard github culture/practice as well.

### CODE REVIEW

**Comment by ilaflott** [comment]
no change to instructions. just breaking up a monolithic `README.md` while I wait for some conda environment cleanup to finish

File: README.md [hunk 1]
```diff
@@ -60,66 +69,6 @@ To be developed:
...
-│   └── /[optional-submodule]
-```
-
-## **Additional Helpful Links**
-* [Official Click Documentation](https://click.palletsprojects.com/en/8.1.x/api/)
-* [`setup.py` Key Words](https://setuptools.pypa.io/en/latest/references/keywords.html)
-* [`meta.yaml` Documentation](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html)
+## ** Contributing to fre-cli **
+
+See [`CONTRIBUTING.md`](https://github.com/NOAA-GFDL/fre-cli/blob/breakup_README/CONTRIBUTING.md)
```
Comment (singhd789) [inline README.md:74]: I think maybe we should put this section above `Further Usage notes`, to keep development kind of together

Comment (singhd789) [reply README.md:74]: that way, users can see how to initialize/start usage (with the methods) and then go into contributing, if they wanted

Comment (ilaflott) [reply README.md:74]: good catch, changing...

Comment (ilaflott) [reply README.md]: How are we looking now?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+├── lazy_group.py
+├── /[tool]
+│   ├── __init__.py
+│   ├── fre[tool].py
+│   ├── README.md
+│   ├── [subCommandScript].py
+│   └── /[optional-submodule]
+```
+
+### Further Usage, Notes
```
Comment (singhd789) [inline CONTRIBUTING.md:60]: Further usage can probably stay in the main README though, no?

**Review by singhd789 — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (ilaflott) [reply CONTRIBUTING.md:60]: now that i read it a little more carefully, two of the bullet points are strictly usage-related... the last one is likely incorrect, and it's concerns addressed via `pip install .` in above instructions.

i'll put it back in the README without the last bullet and remove from CONTRIBUTING.

**Pull request merged**